wicket-gmap3.js

```diff
@@ -380,22 +371,6 @@
                             x: tmp.getAt(0).lng(),
                             y: tmp.getAt(0).lat()
                         });
-
```

Comment:
Here I'm not quite sure about the change, need your input here.
Does the components internally require all rings have the same orientation (clockwise or counter-clockwise)? If not, then maybe we don't need to reverse the inner rings

Reply:
Wicket doesn't care about the orientation of rings but the WKT standard requires that inner rings are reversed with respect to outer rings. For example, if the outer ring is oriented clockwise, then any inner rings should be oriented counter-clockwise.

Reply:
Think this through and update the multiple polygon check